<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Evaluate_dialog</class>
 <widget class="QDialog" name="Evaluate_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>286</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>644</width>
    <height>286</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>644</width>
    <height>286</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>10</y>
     <width>184</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Object-to-Cam (x-y-z-Ry-Rz)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Object-to-Base (x-y-z)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Object-to-Base (Ry)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Object-to-Base (Rz)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Time Calculation</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>18</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Evaluate mode:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>42</y>
     <width>283</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Evaluate</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>77</y>
     <width>277</width>
     <height>190</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Predicted</string>
   </property>
   <widget class="QLineEdit" name="x_predicted">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>21</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>24</y>
      <width>25</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>48</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_predicted">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>48</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>78</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_predicted">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>75</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="rx_predicted">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>102</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>105</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Rx</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ry_predicted">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>129</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>132</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ry</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>159</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Rz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rz_predicted">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>156</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>77</y>
     <width>277</width>
     <height>190</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Reference</string>
   </property>
   <widget class="QLineEdit" name="x_reference">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>21</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>24</y>
      <width>25</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>48</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_reference">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>48</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>78</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_reference">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>75</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="rx_reference">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>102</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>105</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Rx</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ry_reference">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>129</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>132</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ry</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>159</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Rz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rz_reference">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>156</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="time">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>45</y>
     <width>211</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>366</x>
     <y>49</y>
     <width>43</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>x_predicted</tabstop>
  <tabstop>y_predicted</tabstop>
  <tabstop>z_predicted</tabstop>
  <tabstop>rx_predicted</tabstop>
  <tabstop>ry_predicted</tabstop>
  <tabstop>rz_predicted</tabstop>
  <tabstop>x_reference</tabstop>
  <tabstop>y_reference</tabstop>
  <tabstop>z_reference</tabstop>
  <tabstop>rx_reference</tabstop>
  <tabstop>ry_reference</tabstop>
  <tabstop>rz_reference</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
